### ISSUE TITLE
cannot load such file -- ./lib/filestack/ (LoadError)

### ISSUE THREAD

**Issue by jlerpscher** [filestack/filestack-ruby#7]
Hi,

I've got the following error when I am trying to start my rails app:

```
/Users/jle/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/activesupport-5.1.1/lib/active_support/dependencies.rb:292:in `require': cannot load such file -- ./lib/filestack/ (LoadError)
	from /Users/jle/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/activesupport-5.1.1/lib/active_support/dependencies.rb:292:in `block in require'
	from /Users/jle/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/activesupport-5.1.1/lib/active_support/dependencies.rb:258:in `load_dependency'
	from /Users/jle/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/activesupport-5.1.1/lib/active_support/dependencies.rb:292:in `require'
	from /Users/jle/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/filestack-2.0.0/lib/filestack/models/filestack_client.rb:4:in `<top (required)>'
	from /Users/jle/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/activesupport-5.1.1/lib/active_support/dependencies.rb:292:in `require'
	from /Users/jle/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/activesupport-5.1.1/lib/active_support/dependencies.rb:292:in `block in require'
	from /Users/jle/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/activesupport-5.1.1/lib/active_support/dependencies.rb:258:in `load_dependency'
	from /Users/jle/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/activesupport-5.1.1/lib/active_support/dependencies.rb:292:in `require'
	from /Users/jle/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/filestack-2.0.0/lib/filestack.rb:2:in `<top (required)>'
	from /Users/jle/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/bundler-1.15.1/lib/bundler/runtime.rb:82:in `require'
	from /Users/jle/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/bundler-1.15.1/lib/bundler/runtime.rb:82:in `block (2 levels) in require'
	from /Users/jle/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/bundler-1.15.1/lib/bundler/runtime.rb:77:in `each'
	from /Users/jle/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/bundler-1.15.1/lib/bundler/runtime.rb:77:in `block in require'
	from /Users/jle/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/bundler-1.15.1/lib/bundler/runtime.rb:66:in `each'
	from /Users/jle/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/bundler-1.15.1/lib/bundler/runtime.rb:66:in `require'
	from /Users/jle/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/bundler-1.15.1/lib/bundler.rb:108:in `require'
	from /Users/jle/ror/MesPetitsArtistes/config/application.rb:7:in `<top (required)>'
	from /Users/jle/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/spring-2.0.2/lib/spring/application.rb:92:in `require'
	from /Users/jle/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/spring-2.0.2/lib/spring/application.rb:92:in `preload'
	from /Users/jle/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/spring-2.0.2/lib/spring/application.rb:153:in `serve'
	from /Users/jle/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/spring-2.0.2/lib/spring/application.rb:141:in `block in run'
	from /Users/jle/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/spring-2.0.2/lib/spring/application.rb:135:in `loop'
	from /Users/jle/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/spring-2.0.2/lib/spring/application.rb:135:in `run'
	from /Users/jle/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/spring-2.0.2/lib/spring/application/boot.rb:19:in `<top (required)>'
	from /Users/jle/.rbenv/versions/2.3.3/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /Users/jle/.rbenv/versions/2.3.3/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from -e:1:in `<main>'
```

I've simply removed inclusion in `lib/filestack/models/filestack_client.rb:4` and everything seems to be fine.

**Comment by staturecrane** [comment]
@jlerpscher What development environment are you using? Thanks!

**Comment by staturecrane** [comment]
@jlerpscher A patch has been released (2.0.1) addressing this issue. Thank you for alerting us. Closing this issue now.
